src/QueueableCommand.php

```diff
@@ -3,12 +3,11 @@
 namespace League\Tactician\Bernard;
 
 use Bernard\Message;
-use League\Tactician\Command;
 
 /**
  * Creates a common interface for queueable commands
  */
-interface QueueableCommand extends Command, Message
+interface QueueableCommand extends Message
```

Comment:
If we ever want to add back an interface like this (and it extends `Bernard\Message`) then we are free to do that without breaking BC. Since there is no type hinting (only an instance check which doesn't matter in this case, because the interface would extend Bernard\Message). We can always add some magic in the code.